Sources/SpeziAccountPhoneNumbers/SpeziAccountPhoneNumbers.docc/SpeziAccountPhoneNumbers.md

```diff
@@ -34,6 +34,41 @@ var configuredValues: AccountValueConfiguration {
 }
 ```
 
+1.1 (Optional) Configure custom encoding/decoding strategy for SpeziFirebase storage:
```

Comment:
I would suggest to make the general explanation independent of Firebase and the `FirestoreAccountStorage` example. Maybe let's start with a general motivation and then transition to the example that says "hey, if you, e.g., use Firebase, here is an example:"